api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -143,10 +159,13 @@ public void createRuntime(
     leonardoRetryHandler.run(
         (context) -> {
           runtimesApi.createRuntime(
-              googleProject,
-              runtimeName,
+              runtime.getGoogleProject(),
```

Comment:
This code still needs to function without the new feature flag enabled. In this current version, I don't think it will function either way.

Reply:
I can add switches for the feature flag but it does work as is. I updated the UI's API call to send over default-dataproc as the configuration type.

Reply:
ack - this is obsolete, this was also because I missed the logic in RuntimeController

Reply:
hm, in that case, is it OK to not feature flag this? We would be losing out of the ability to toggle it off if there's a bug but functionality wise, it should be the same if its on or off.

Reply:
Yeah I suppose machine type and disk size could be hardcoded on the ui instead with this change. The feature flag could then just control whether to look at the clusteroverride

Reply:
Yeah I suppose machine type and disk size could be hardcoded on the ui instead with this change. The feature flag could then just control whether to look at the clusteroverride

Reply:
Or else, just fallback to the previous config value if the client does not supply a runtime config; should be required with the new flag though. I'm not sure which will be easier here.